sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/CqlDataWriterDAO.java

```diff
@@ -239,6 +240,12 @@ public void purgeUnsafe(Table table) {
         _astyanaxWriterDAO.purgeUnsafe(table);
     }
 
+    private void checkError(AtomicReference<Throwable> t) {
+        if (t.get() != null) {
+            Throwables.propagate(t.get());
```

Comment:
This is really minor, but usually this is written as:

`throw Throwables.propagate(t.get());`

Without the leading "throw" the IDE will warn that the throwable is not thrown, even though logically that code is never reached.